sys/dev/hyperv/pcib/vmbus_pcib.c

```diff
@@ -50,7 +50,7 @@
 #include <vm/pmap.h>
 
 #if defined(__aarch64__)
-#include <arm64/include/intr.h>
+#include <machine/intr_machdep.h>
```

Comment:
Wow.  Just wow.  The qualities of this are just...

Alas, I'm quite unsure why the original author did things this way.  On further consideration, perhaps this should simply be `#include <arm64/include/intr_machdep.h>` to fulfill the minimum.  Let someone involved with this do the proper job.

Reply:
This comes from 6e5b082cab8d9dd0eb0bce65cf8cf268f11c7a25.  Looks like they wanted ac29ddabea3fedaace5c7dab54287ee2edd203a7 (or the inverse of `intr_machdep.h` => `intr.h`) as well.  Alas the qualities of that implementation are poor.